lib/python-parser/metadata-parser.ts

```diff
@@ -29,7 +30,12 @@ export function getPackageInfo(fileContent: string): PythonPackage {
       }
     }
   }
-  return { name: name.toLowerCase(), version, dependencies } as PythonPackage;
+  const validVersion = semver.coerce(version);
+  return {
+    name: name.toLowerCase(),
+    version: validVersion ? validVersion.toString() : version,
```

Comment:
Does it make sense to fallback to the original version if the semver library can't parse it? 
Or should we potentially throw an error instead? What's the failure mode here? :)

Reply:
That sounds to me like you should throw a (custom?) error and handle that case in the calling code to discard this file? 😄

Reply:
I guess that's the JS way :P

Reply:
Both ways work :)

Reply:
I went with your suggestion